ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>Add here that the setup notebook is used to bring up rotator and mount/CCW</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated in the commit: 137c4fd to f4b262b already. There is the case-sensitive error in your folder and I could not reach the LSP at summit and Tucson Teststand to rewrite the history. Stay like this at this time. Thanks!